src/wfpt.pyx

```diff
@@ -70,6 +75,166 @@ def wiener_like(np.ndarray[double, ndim=1] x, double v, double sv, double a, dou
     return sum_logp
 
 
+def wiener_like_rlddm(np.ndarray[double, ndim=1] x,
+                      np.ndarray[long, ndim=1] response,
+                      np.ndarray[double, ndim=1] feedback,
+                      np.ndarray[long, ndim=1] split_by,
+                      double q, double alpha, double dual_alpha, double v, 
+                      double sv, double a, double z, double sz, double t,
+                      double st, double err, int n_st=10, int n_sz=10, bint use_adaptive=1, double simps_err=1e-8,
+                      double p_outlier=0, double w_outlier=0):
+    cdef Py_ssize_t size = x.shape[0]
+    cdef Py_ssize_t i
+    cdef Py_ssize_t s_size
+    cdef int s
+    cdef double p
+    cdef double sum_logp = 0
+    cdef double wp_outlier = w_outlier * p_outlier
+    cdef double alfa
+    cdef double neg_alpha = (2.718281828459**alpha) / (1 + 2.718281828459**alpha)
+    cdef double pos_alpha = 2.718281828459**(alpha + dual_alpha) / (1 + 2.718281828459**(alpha + dual_alpha))
+    cdef np.ndarray[double, ndim=1] feedbacks = np.empty(0, dtype=np.double)
```

Comment:
I think you can just remove all of these which you are redefining in the loop.

Reply:
Moved the neg_alpha and pos_alpha to just be defined in the for-loop. I've tried defining arrays (e.g. feedbacks) with cdef directly inside the for-loop, but that causes an error.

Reply:
And you can remove the ` = np.empty()` for everything here, except `qs` (and `unique`).

Reply:
OK, just remove the empty's and it should be fine.